applications/luci-app-packet-capture/root/usr/sbin/packet_capture

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env lua
```

Comment:
I believe this and the subsequent helper executables or not meant to be directly usable on the shell as normal commands, so please move them into `/usr/libexec/` to not pollute `$PATH`. Also I think all these scripts should be merged into one single executable.

Reply:
I moved scripts to libexec. One executable is a good idea but if you agree then I would like to keep separate scripts for now. Merging scripts sh and lua into one robust script might require some additional refactoring here and there so if possible then I would like to let users try this application. I promise that I will merge it into one exec in the nearest feature. More, I am going to push iperf3 application that is similar to the packet capture application. Iperf3 application will have a single executable. After that, I will copy the same structure to the packet capture application.